molecule/default/molecule.yml

```diff
@@ -0,0 +1,26 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+  # name: podman
```

Comment:
I can't get docker to work on my laptop in the same way that it works on github; i wonder if anybody's 'molecule test' just works with their docker?